mutiny-core/src/nodemanager.rs

```diff
@@ -926,13 +926,19 @@ impl<S: MutinyStorage> NodeManager<S> {
         let onchain = self.list_onchain()?;
         let mut activity = Vec::with_capacity(lightning.len() + onchain.len());
         for ln in lightning {
-            activity.push(ActivityItem::Lightning(Box::new(ln)));
+            // Only show paid invoices
+            if ln.paid {
+                activity.push(ActivityItem::Lightning(Box::new(ln)));
+            }
```

Comment:
I think it would be valid to show pending invoices that have not been paid yet.

Reply:
It would be nice but I'm not sure how to do that. Rn what happens is you create an invoice and an on-chain addy. If it gets paid on-chain you have the invoice in the activity list as well.

Reply:
Can work toward this in a followup? Def would be good to solve if we can find a nice solution

Reply:
Yeah can be another feature request if we really want it.